packages/wonder-blocks-tooltip/docs.md

```diff
@@ -0,0 +1,9 @@
+Tooltips should be used to help describe an on screen object.
+
+Tooltips:
+
+- contain text
+- may contain small graphic elements to complement the text
+- do not include actions
```

Comment:
"are dismissed when focus/hover moves away" or something like that.

Reply:
I took this text directly from the Zeplin board. Definitely needs fleshing out a bit more.